swagger.yml

```
@@ -722,6 +722,8 @@ paths:
   /purge:
     post:
       operationId: purgeCache
+      description: >-
+        Purges cached content from Netlify's CDN. Support purging by Cache-Tags.
```

[pebble] suggestion to tidy up grammar and match response header name verbatim for clarity

```suggestion
Purges cached content from Netlify's CDN. Supports purging by Cache-Tag.
```